### PR TITLE
Add errors property to submissionError

### PR DESCRIPTION
**Pull request by waynebrantley**
@iamandrewluca small missing property, has been around for a long time.

### CODE REVIEW

File: types/redux-form/lib/SubmissionError.d.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { FormErrors } from "../index";
 
 export declare class SubmissionError<FormData = {}, ErrorType = string> extends Error {
   constructor(errors?: FormErrors<FormData, ErrorType>);
+  public errors: FormErrors<FormData, ErrorType> | undefined;
```
Comment (bwlt) [inline types/redux-form/lib/SubmissionError.d.ts]: [Travis reported](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/569426943#L369): `member-access  'public' is implicit.` Please fix.
Also in my opinion it should not be nullable: from [source code](https://github.com/erikras/redux-form/blob/master/src/SubmissionError.js) the `errors` member is always set up.

**Review by bwlt — CHANGES_REQUESTED** [review]

File: types/redux-form/lib/SubmissionError.d.ts (hunk 1 above)
Comment (iamandrewluca) [reply types/redux-form/lib/SubmissionError.d.ts]: In this case the constructor maybe also needs fixing.

Comment (waynebrantley) [reply types/redux-form/lib/SubmissionError.d.ts]: I made the changes requested.  The 'public' was required when I did it locally be linter/typescript - however, I removed it as requested.

**Review by unrevised6419 — APPROVED** [review]

**Review by bwlt — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Please create PRs using the template in the future to help make the review process easier.

Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**